### PR TITLE
Add a "debug" command

### PR DESCRIPTION
**Pull request by lyrixx**
image

### CODE REVIEW

**Review by pyrech — APPROVED** [review]
I wonder whether the debug command should be hidden or not?

**Comment by lyrixx** [comment]
> I wonder whether the debug command should be hidden or not?

Good question ! I have removed it from the test for now.

**Comment by lyrixx** [comment]
The command is now hidden, with an alias to `castor:debug`.

File: src/Console/Command/DebugCommand.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        private readonly ContextRegistry $contextRegistry,
+    ) {
+        parent::__construct();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $tables = [
```
Comment (welcoMattic) [inline src/Console/Command/DebugCommand.php:33]:
```suggestion
        $table = [
```
and usages

**Review by pyrech — APPROVED** [review]

**Pull request merged**